### ISSUE TITLE
Improve reporting for DataDog heading violation metrics.

### ISSUE THREAD

**Issue by ndouglas** [department-of-veterans-affairs/va.gov-cms#12627]
## Description
Just need to tweak the metric names a tad.

## Acceptance Criteria
- [ ] Metric names are reported correctly and not stomping all over a global namespace.
